docs/ibc-relayer.md

```diff
@@ -0,0 +1,78 @@
+   - This is defined by the `checkpoint_finalization_timeout` parameter of [x/btccheckpoint module](https://github.com/babylonlabs-io/babylon/blob/main/x/btccheckpoint/README.md)
+   - Any unbonding requests from that checkpointed epoch are then matured
+   - Given Bitcoin's average block time of ~10 minutes, the average unbonding time is about 50 hours
+
+3. **IBC Light Client Configuration**:
+   - IBC light clients for Babylon Genesis on other chains should have a lower trusting period (~33 hours)
+   - This is about 2/3 of the unbonding period, following standard IBC security practices
+   - This configuration only affects light clients of Babylon Genesis on other chains
+   - The trusting period of other chains' light clients on Babylon Genesis remains unaffected
+
+Due to these unique characteristics, special attention is required when configuring the relayer's trusting period and client refresh rate.
+
+## Prerequisites
+
+Before beginning, ensure you have:
+1. Rust installed and configured
+2. Hermes installed (refer to [Hermes Quick Start](https://hermes.informal.systems/quick-start/) for installation steps)
+3. Access to RPC and gRPC endpoints for both chains
+4. Wallets funded with native tokens for both chains
+
+## Relayer Configuration
+
+When setting up a relayer for Babylon, pay special attention to these parameters:
+
+1. **Trusting Period**: Should be set to approximately 2/3 of Babylon's unbonding period
+   - Babylon's unbonding period is ~50 hours (based on ~300 BTC blocks)
+   - Therefore, the trusting period should be set to ~33 hours
+
+2. **Client Refresh Rate**: A higher refresh rate is recommended (1/5 of trusting period, i.e., ~6.6 hours)
+   - Make sure `refresh = true` is set in the configuration
+
+**Important**: Do not use the default 21-day unbonding period that Hermes might fetch from the `x/staking` module query. Always set the trusting period based on Babylon's actual unbonding period of ~50 hours.
+
+For complete setup instructions, including wallet configuration, connection setup, and channel creation, refer to:
+- [Celestia's IBC Relayer Guide](https://docs.celestia.org/how-to-guides/ibc-relayer)
+- [Osmosis's Relayer Guide](https://docs.osmosis.zone/osmosis-core/relaying/relayer-guide)
+
+## Monitoring and Maintenance
+
+Regular monitoring of your IBC clients is crucial. Monitor the `client_updates_submitted_total` metric, which counts the number of client update messages submitted between chains. This metric should increase over time as your relayer submits updates to keep the IBC clients synchronized. For detailed information about this metric, refer to [Hermes metrics documentation](https://hermes.informal.systems/documentation/telemetry/operators.html#what-is-the-overall-ibc-status-of-each-network).
+
+For network-wide health monitoring, visit [Babylon Testnet Network Metrics](https://insights.informal.systems/babylon-testnet-5/network-metrics).
```

Comment:
```suggestion
For network-wide health monitoring, visit the [Babylon Testnet Network Metrics](https://insights.informal.systems/babylon-testnet-5/network-metrics).
```

What are these metrics and why are we referencing them? They seem to relate to the testnet. I would avoid mentioning those here.